src/node/stage/approved.rs

```diff
@@ -236,6 +261,11 @@ impl Approved {
         self.vote_for_event(AccumulatingEvent::User(event));
     }
 
+    /// Is the node with the given id an elder in our section?
+    pub fn is_our_elder(&self, id: &PublicId) -> bool {
+        self.shared_state.sections.our().is_member(id)
```

Comment:
I think there is function called `is_self_elder` shall do the job?
`is_member` seems could be an adult or infant?

Reply:
`is_self_elder` is removed. `EldersInfo::is_member` is now renamed to `contains_elder` to avoid confusion.